README.md

```diff
@@ -29,19 +29,25 @@ HexChat developers decided that their script should focus on their specific need
     pacman -S nasm patch gettext make coreutils diffutils wget yasm pkg-config
     ```
 
-1. Clone [this repository](https://github.com/wingtk/gtk-win32) to _C:\gtk-build\github\gtk-win32_ It contains the build script, project files and patches.
+1. Clone [this repository](https://github.com/wingtk/gvsbuild) to _C:\gtk-build\gvsbuild_ It contains the build script, project files and patches.

```

Comment:
IMO, it is important to have precise instructions, especially for people who not experienced with git. Old instructions left unclear if the git clone will create github folder as well. That's why I would prefer the steps to be more precise this time.